### DISCUSSION TITLE
Hclustering Distance

### DISCUSSION THREAD

**Discussion by KElisavet** [hnolCol/instantclue#89] [Q&A]
Hello, I would like to ask how I can export the cluster distances from hierarchical clustering.
Thank you!

**Comment by hnolCol** [comment]
Hi, I assume you are talking about the distance matrix, right? That will have the dimension: 

rows x rows for the row clustering and columns x columns for the top clustering. 

At the moment there is no option to access the numeric data. I have implement it as such that you can access the data via the graph context menu:
<img width="454" alt="image" src="https://github.com/user-attachments/assets/345f2497-cd73-47b4-8f98-0fac1b6beeb8">

Is this what you need? I will double-check and then implement it in the new version!
